edxval/serializers.py

```diff
@@ -79,8 +79,9 @@ def to_native(self, value):
 
     def from_native(self, data):
         if data:
-            return CourseVideo(course_id=data)
-
+            course_video = CourseVideo(course_id=data)
+            course_video.full_clean(exclude=["video"])
```

Comment:
Without this, the course_id is never actually validated. Specifically, ImportTest.test_invalid_course_id fails without this.